package.json

```diff
@@ -1,14 +1,17 @@
 {
-  "name": "compound-protocol-v2.5",
+  "name": "compound-iii",
   "version": "0.1.0",
   "description": "",
   "main": "dist/index.js",
   "files": [
-    "dist"
+    "dist",
+    "deployments/**/roots.json"
   ],
   "exports": {
-    ".": "./dist/index.js",
-    "./*": "./dist/*.js"
+    ".":  "./dist/index.js",
+    "./*": "./dist/*.js",
+    "./kovan/roots.json": "./deployments/kovan/roots.json",
```

Comment:
Annoyingly, you cannot specify a glob for a directory; this does not work:

```
"./**/roots.json": "./deployments/**/roots.json"
```

Reply:
We might be able to reduce some of this duplication by exporting the entire `deployments/` directory, instead of each roots file